common/src/main/java/org/jboss/hal/testsuite/creaper/ResourceVerifier.java

```diff
@@ -236,6 +236,85 @@ public ResourceVerifier verifyAttributeIsUndefined(String attributeName) throws
         return verifyAttributeIsUndefined(attributeName, null);
     }
 
+    private boolean isModelNodePresentInListAttributeValue(String attributeName, ModelNode value) throws IOException {
+        final ModelNodeResult modelNodeResult = ops.readAttribute(resourceAddress, attributeName);
+        return modelNodeResult.listValue().stream()
+                .peek(modelNode -> log.trace("Comparing '{}' with list member '{}'.", modelNode.toString(), value.toString()))
+                .anyMatch(modelNode -> modelNode.equals(value));
+    }
+
+    /**
+     * Verifies that list type attribute contains give value.
+     * @param value Value which should be present in the list.
+     */
+    public ResourceVerifier verifyListAttributeContainsValue(String attributeName, ModelNode value, String errorMessageSuffix)
+            throws Exception {
+        waitFor(() -> {
+            ModelNodeResult actualResult = ops.readAttribute(resourceAddress, attributeName);
+            return actualResult.isSuccess() &&
+                    actualResult.hasDefined(Constants.RESULT) &&
+                    isModelNodePresentInListAttributeValue(attributeName, value);
+        });
+
+        final ModelNodeResult modelNodeResult = ops.readAttribute(resourceAddress, attributeName);
+        modelNodeResult.assertSuccess();
+
+        Assert.assertTrue("Given value '" + value.toString() + "' is not present in list attribute '" + attributeName + "'!" +
+                        (errorMessageSuffix == null || errorMessageSuffix.isEmpty() ? "" : " " + errorMessageSuffix),
+                isModelNodePresentInListAttributeValue(attributeName, value));
+
+        return this;
+    }
+
+
+
+    /**
+     * Verifies that list type attribute contains give value.
+     * @param value Value which should be present in the list.
+     */
+    public ResourceVerifier verifyListAttributeContainsValue(String attributeName, ModelNode value) throws Exception {
+        return verifyListAttributeContainsValue(attributeName, value, null);
+    }
+
+    /**
+     * Verifies that list type attribute contains give value.
+     * @param value Value which should be present in the list.
+     */
+    public ResourceVerifier verifyListAttributeContainsValue(String attributeName, String value) throws Exception {
+        return verifyListAttributeContainsValue(attributeName, new ModelNode(value));
+    }
+
+    /**
+     * Verifies that list type attribute contains give value.
+     * @param value Value which should be present in the list.
+     */
+    public ResourceVerifier verifyListAttributeDoesNotContainValue(String attributeName, ModelNode value, String errorMessageSuffix)
+            throws Exception {
+        waitFor(() -> {
+            ModelNodeResult actualResult = ops.readAttribute(resourceAddress, attributeName);
+            return actualResult.isSuccess() &&
+                    actualResult.hasDefined(Constants.RESULT);
```

Comment:
I suggest to add
`&& !isModelNodePresentInListAttributeValue(attributeName, value)`
to the condition.